docker/Dockerfile-base

```diff
@@ -38,6 +38,13 @@ RUN git lfs install --skip-repo && \
     # The base image ships with `pydantic==1.8.2` which is not working
     pip3 install -U --no-cache-dir pydantic==1.10.10
 
-RUN if [ "$PYTORCH_VERSION" = "2.7.0" ] ; then \
+RUN if [ "$TORCH_CUDA_ARCH_LIST" = "9.0+PTX" ] ; then \
+        git clone https://github.com/Dao-AILab/flash-attention.git; \
+        cd flash-attention; \
+        git checkout v2.7.4.post1; \
```

Comment:
I saw that this release is Jan 2025 and there's been a lot of changes to `hopper` dir since then.